src/app/shared/components/category-navigation/category-navigation.component.html

```diff
@@ -23,7 +23,7 @@
         <div
           [matTooltip]="category.disabled && !inactive ? ('AVAILABLE_SOON' | translate) : undefined"
           matTooltipClass="tooltip-white"
-          matTooltipPosition="before"
+          matTooltipPosition="below"
```

Comment:
Tooltip pod spodem, pojawi się bez "dziubka" strzałki, można dopisać css tak jak dal pozycji left, right

Reply:
A czy ten dziubek jest w ogóle potrzebny? z default Dziubka w toultipach materiałowych chyba nie ma?

Reply:
Ja się nie upieram :) ale na makietach był.

Reply:
na makietach był również w złym miejscu :) jak się przyczepią to osobiście poprawię